app/views/docs/textarea_view.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class Docs::TextareaView < ApplicationView
+  def view_template
+    div(class: "max-w-2xl mx-auto w-full py-10 space-y-10") do
+      render Docs::Header.new(title: "Textarea", description: "Displays a textarea field.")
+
+      TypographyH2 { "Usage" }
+
+      render Docs::VisualCodeExample.new(title: "Textarea", context: self) do
+        <<~RUBY
+          div(class: "grid w-full max-w-sm items-center gap-1.5") do
+            Textarea(placeholder: "Textarea")
+          end
+        RUBY
+      end
+
+      render Docs::VisualCodeExample.new(title: "Disabled", context: self) do
+        <<~RUBY
+          div(class: "grid w-full max-w-sm items-center gap-1.5") do
+            Textarea(disabled: true, placeholder: "Disabled")
+          end
+        RUBY
+      end
+
+      render Docs::VisualCodeExample.new(title: "With FormField", context: self) do
+        <<~RUBY
+          div(class: "grid w-full max-w-sm items-center gap-1.5") do
+            FormField do
+              FormFieldLabel(for: "textarea") { "Textarea" }
+              FormFieldHint { "This is a textarea" }
+              Textarea(placeholder: "Textarea", id: "textarea")
+              FormFieldError()
+            end
+          end
+        RUBY
+      end
+
+      render Docs::ComponentsTable.new(components)
+    end
+  end
+
+  private
+
+  def components
+    [
+      Docs::ComponentStruct.new(name: "Textarea", source: "lib/rbui/textarea/textarea.rb", built_using: :phlex)
```

Comment:
source fixed
<img width="1180" alt="image" src="https://github.com/user-attachments/assets/cc163b69-9c61-4f75-a3fe-de12b579dd3a">

Reply:
Last update:
The views need to have this updated structure:
https://github.com/rbui-labs/rbui-web/pull/119/files#diff-91e4c03b6dc05c5672ab6eabbff23ed3bd234bc13e085b891e5ed52e91e17c16

This line will register all the code to the parser to extract the components being used and also show the files that the component has:
https://github.com/rbui-labs/rbui-web/blob/8431fdf1ff0feb45c0aa1643c6175f79d7944caa/app/views/docs/alert_dialog_view.rb#L30